### DISCUSSION TITLE
3版本 通过fid获取群公告的时候 出现错误

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#78] [打扰了]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

 @Listener
    public void onEvent(MiraiGroupMessageEvent event) {
        MiraiGroup group = event.getGroup();

        // 获取群管理器
        MiraiAnnouncements announcements = group.getAnnouncements();
        

       // 这个地方出现我无法解决问题 
     // Field 'msg' is required for type with serial name 'net.mamoe.mirai.internal.contact.announcement.GroupAnnouncement', but it was missing
        MiraiOnlineAnnouncement blocking = announcements.getBlocking("36ee6e2800000000742cc963d7e90900"); 

        List<MiraiMember> miraiMembers = blocking.membersBlocking(true);

        for (MiraiMember miraiMember:miraiMembers){
            System.out.println(miraiMember);
        }
    }

**Comment by ForteScarlet** [comment]
异常相关内容请通过 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml) 反馈
